src/hotspot/cpu/aarch64/interp_masm_aarch64.cpp

```diff
@@ -1833,3 +1833,15 @@ void InterpreterMacroAssembler::profile_parameters_type(Register mdp, Register t
     bind(profile_continue);
   }
 }
+
+void InterpreterMacroAssembler::load_resolved_indy_entry(Register cache, Register index, Register tmp) {
+  // Get index out of bytecode pointer, get_cache_entry_pointer_at_bcp
+  get_cache_index_at_bcp(index, 1, sizeof(u4));
+  // Get address of invokedynamic array
+  ldr(cache, Address(rcpool, in_bytes(ConstantPoolCache::invokedynamic_entries_offset())));
+  // Scale the index to be the entry index * sizeof(ResolvedInvokeDynamicInfo)
+  mov(tmp, sizeof(ResolvedIndyEntry));
+  mul(index, index, tmp);
```

Comment:
On 64bits platform, sizeof(ResolvedIndyEntry) is 16, a power of two, so shift instruction could be used instead of a multiply instructions (with an assert in case the size of ResolvedIndyEntry is changed).